core-bundle/contao/drivers/DC_Table.php

```diff
@@ -3912,6 +3917,7 @@ protected function treeView()
 			if (!($GLOBALS['TL_DCA'][$this->strTable]['config']['notCopyable'] ?? null))
 			{
 				$arrButtons['copy'] = '<button type="submit" name="copy" id="copy" class="tl_submit" accesskey="c">' . $GLOBALS['TL_LANG']['MSC']['copySelected'] . '</button>';
+				$arrButtons['copyMultiple'] = '<button type="submit" name="copyMultiple" id="copyMultiple" class="tl_submit" accesskey="c">' . $GLOBALS['TL_LANG']['MSC']['copyMultiple'] . '</button>';
```

Comment:
We have to change `accesskey="c"` to something that is not yet in use. (The same applies to line 4867.)